### PR TITLE
gmt 6.1.0

### PR DESCRIPTION
**Pull request by seisman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gmt.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.0.0-src.tar.xz"
-  mirror "https://fossies.org/linux/misc/GMT/gmt-6.0.0-src.tar.xz"
-  sha256 "8b91af18775a90968cdf369b659c289ded5b6cb2719c8c58294499ba2799b650"
-  revision 7
+  url "https://github.com/GenericMappingTools/gmt/releases/download/6.1.0/gmt-6.1.0-src.tar.xz"
+  mirror "ftp://ftp.soest.hawaii.edu/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://fossies.org/linux/misc/GMT/gmt-6.1.0-src.tar.xz"
+  sha256 "ad02780153c53a1116ae0cc7945b6f533f066af44c30d7f95ff138cfede1867c"
+  revision 0
```
Comment (dawidd6) [inline Formula/gmt.rb:9]:
```suggestion
```

Comment (seisman) [reply Formula/gmt.rb]: Done.

File: Formula/gmt.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   homepage "https://www.generic-mapping-tools.org/"
-  url "https://github.com/GenericMappingTools/gmt/releases/download/6.0.0/gmt-6.0.0-src.tar.xz"
-  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.0.0-src.tar.xz"
-  mirror "https://fossies.org/linux/misc/GMT/gmt-6.0.0-src.tar.xz"
-  sha256 "8b91af18775a90968cdf369b659c289ded5b6cb2719c8c58294499ba2799b650"
-  revision 7
+  url "https://github.com/GenericMappingTools/gmt/releases/download/6.1.0/gmt-6.1.0-src.tar.xz"
+  mirror "ftp://ftp.soest.hawaii.edu/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://fossies.org/linux/misc/GMT/gmt-6.1.0-src.tar.xz"
```
Comment (chenrui333) [inline Formula/gmt.rb:7]: How do we need three mirrors instead of two?

Comment (seisman) [reply Formula/gmt.rb]: Two mirrors are enough. The first one is the GMT official FTP server, so I just removed the third mirror.

Comment (chenrui333) [reply Formula/gmt.rb]: 👍

**Review by chenrui333 — APPROVED** [review]
lgtm, thanks!

**Pull request closed without merging**